### PR TITLE
Update README instructions for simulation

### LINKED ISSUE #173: Adjust simulation script to enable easier demo
Allow the "requestDate" to be some time in the past so its easier to simulate reasonable wait times.

Document good parameters to use

### PR DESCRIPTION
**Pull request by mscharfstein**
Fixes #173 

> It's a good idea to open an issue first for discussion.

- [ ] Tests pass
- [ ] Appropriate changes to README are included in PR
- [ ] Feature has been fleshed out in design document

### CODE REVIEW

File: simulation/producer/README.md [hunk 1]
```diff
@@ -18,10 +18,11 @@ where the api-url is the URL (host + port) where the Call Me Back API is running
 There are other optional flags:
   * --calls = the number of calls that should be sent during each period; defaults to 10
   * --period = the length of time (in minutes) during which the specified number of calls are sent; defaults to 60 minutes. This represents the wall clock time, and can be sped up by the --speed-up flag.
-  * NOT READY TO BE USED: --speed-up = the factor at which wall clock time should be sped up for the simulation. For example, if the period is 60 minutes, and the speed-up factor is 60, the simulation would run 60 times faster than real time and would send requests every minute, instead of every 60 minutes. Defaults to 1, which means it will be run in real time.
+  * --rewind-request-dates-by = the number of minutes the calls' request dates should be rewinded from the present time; default to 0 minutes (wall clock time). For example, if this is 30, the request date for each call will be
```
Comment (egracer) [inline simulation/producer/README.md:21]: Clever solution to the wall clock time problem 👍

File: simulation/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Details about running the two scripts are outlined in the consumer and producer READMEs.
+
+## Simulation Parameters
+
+Both scripts take parameters to indicate how quickly calls should be added and removed from the queue. The
+following outlines some parameters that can be used to explore wait times in a meaningful way.
+
+1) Seed the database with a history of resolved calls to have a starting point for wait time calculations:
+    a) Run producer script with --calls 10 --rewind-request-dates-by 60. This will add 20 calls that all have request dates 60 minutes before the current time. Once this completes, ctrl+C to stop the script.
```
Comment (kebrahim) [inline simulation/README.md:19]: isn't it adding 10 calls, not 20?

File: simulation/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    b) Run consumer script with --period 0.1 --length .5. This will relatively quickly pick up the calls in the queue (one every 6 seconds) and then resolve each call after 30 seconds. Once this completes, ctrl+C to stop the script.c) Once a) and b) complete, the database will have 10 calls with wait times around an hour each.
```
Comment (kebrahim) [inline simulation/README.md:20]: put "c)" on a separate line

File: simulation/README.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+2) Make a reservation in the UI. The wait time should be around an hour given the seeded data in the database.
```
Comment (kebrahim) [inline simulation/README.md:21]: both naive & moving average wait times?

File: simulation/README.md (hunk 2 above)
Comment (kebrahim) [inline simulation/README.md:19]: these aren't indenting in the actual README. is that possible?

Comment (mscharfstein) [reply simulation/README.md:19]: yes sorry :)

File: simulation/README.md (hunk 4 above)
Comment (mscharfstein) [reply simulation/README.md:21]: yes, to start. updated wording

File: simulation/README.md (hunk 2 above)
Comment (mscharfstein) [reply simulation/README.md:19]: should be fixed now

**Pull request merged**